tools/importers/helpers/tokensReader.ts

```diff
@@ -46,10 +46,13 @@ export const recursiveTokensReader = (obj, keyResult: string): Array<ITypography
 			}
 		}
 
-		let token = keyResult
-			.replace(/([a-z0-9])([A-Z])/g, '$1-$2')
+		let tokenSplit = keyResult
+			// .replace(/([a-z0-9])([A-Z])/g, '$1-$2')
 			.replace(/\s+/g, '-')
-			.toLowerCase();
+			.split('-');
+		let tokenUsage = tokenSplit.shift() + '';
```

Comment:
why do we add an empty string here?

Reply:
Can't use `toString()` becase of error `Object possibly 'undefined'` so I convert to string in the different way.

Reply:
IMO `string.split('-').shift()` always return a string, so why do we need to cast it?

Reply:
I know, but array can be empty, and after shift we will have undefined element, which is put to `camelize` function where parameter must be a string. So I need 
- cast to string OR
-  add | undefined  type as parameter in camelize OR
- do `if` checking type, which will be always true in our case.

Reply:
poor TS...